lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -63,7 +65,15 @@ def create_client_properties
 
         return if file_found?(client_properties_path)
 
-        content = <<~CLIENT_PROPERTIES
+        algorithm = server_host_is_ipaddr? ? "" : "HTTPS"
+        content = secure? ? secure_client_properties_content(algorithm) : unsecure_client_properties_content(algorithm)
+
+        File.write(client_properties_path, content)
+      end
+
+      def secure_client_properties_content(algorithm)
```

Comment:
Can this method leverage the unsecure method for the majority of the content?

Reply:
I'd be glad to discuss suggestions you may have but it took a lot of work to get the content of the client properties file content nailed down for the matrix of `hostname` vs `IP Addr` and `secure` vs `unsecure`.  I think leaving it as two separate fully complete methods may help future maintainers.

Reply:
I think keeping them separate is fine for now.  I can see both side of the argument.

Reply:
@bdunne This is resolved by commit: https://github.com/ManageIQ/manageiq-appliance_console/pull/149/commits/14e2e666ddcd3b1c1cbebadc3334105853b954e3